plugins/performance-lab/includes/site-health/avif-headers/helper.php

```diff
@@ -0,0 +1,85 @@
+<?php
+/**
+ * Helper function to detect and test AVIF header information.
+ *
+ * @package performance-lab
+ * @since n.e.x.t
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Callback for avif_headers test.
+ *
+ * @since n.e.x.t
+ *
+ * @return array{label: string, status: string, badge: array{label: string, color: string}, description: string, actions: string, test: string} Result.
+ */
+function avif_headers_check_avif_headers_test(): array {
+	$result = array(
+		'label'       => __( 'Your site sends AVIF image headers', 'performance-lab' ),
+		'status'      => 'good',
+		'badge'       => array(
+			'label' => __( 'Performance', 'performance-lab' ),
+			'color' => 'blue',
+		),
+		'description' => sprintf(
+			'<p>%s</p>',
+			esc_html__(
+				/* translators: 1: image/avif, 2: content-type */
+				sprintf(
+					'Images with the mime type %1$s served without the correct %2$s header may not render properly.',
+					'image/avif',
+					'content-type'
+				),
+				'performance-lab'
+			)
```

Comment:
I believe this has to be rearranged a bit:

```suggestion
			esc_html(
				sprintf(
					/* translators: 1: image/avif, 2: content-type */
					__( 'Images with the mime type %1$s served without the correct %2$s header may not render properly.', 'performance-lab' ),
					'image/avif',
					'content-type'
				)
			)
```